repository/kafka/docs/latest/release-notes.md

```diff
@@ -2,12 +2,13 @@
 
 ## latest
 
-- Add parameter `RUN_USER_WORKLOAD` to create some dummy user workload over kafka service.
-
 ## v1.0.2
 
 - Add parameter `KERBEROS_USE_TCP` to use TCP protocol for KDC connection
 - Add parameter `ADD_SERVICE_MONITOR` to create a service monitor for KUDO Kafka cluster.
+- Enabled external KUDO Kafka access.
+- Add parameter `RUN_USER_WORKLOAD` to create some dummy user workload over kafka service.
+- Add disk usage metrics
```

Comment:
are there any docs on these?

Reply:
added to the `monitoring.md` and the dashboard preview was updated